### PR TITLE
OSDOCS-12570: CAS scaleup limitations

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12+

Issue:
[OSDOCS-12570](https://issues.redhat.com//browse/OSDOCS-12570)

Link to docs preview:
[About the cluster autoscaler](https://86239--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling.html#cluster-autoscaler-about_applying-autoscaling)

QE review:
- [x] QE has approved this change.

Additional information:
Added note per ticket, added subheadings to better organize the "About" section

### CODE REVIEW

File: modules/cluster-autoscaler-about.adoc [hunk 1]
```diff
@@ -55,14 +62,22 @@ If you configure the cluster autoscaler, additional usage restrictions apply:
...
 
 The cluster autoscaler takes pod priorities into account. The Pod Priority and Preemption feature enables scheduling pods based on priorities if the cluster does not have enough resources, but the cluster autoscaler ensures that the cluster has resources to run all pods. To honor the intention of both features, the cluster autoscaler includes a priority cutoff function. You can use this cutoff to schedule "best-effort" pods, which do not cause the cluster autoscaler to increase resources but instead run only when spare resources are available.
 
 Pods with priority lower than the cutoff value do not cause the cluster to scale up or prevent the cluster from scaling down. No new nodes are added to run the pods, and nodes running these pods might be deleted to free resources.
 
-Cluster autoscaling is supported for the platforms that have machine API available on it.
```
Comment (jeana-redhat) [inline modules/cluster-autoscaler-about.adoc:64]: The MAPI requirement is already stated in the assembly

**Review by elmiko — COMMENTED** [review]
/lgtm

**Comment by stevsmit** [comment]
LGTM!

**Pull request merged**